### PR TITLE
Switch back to whatwg-fetch

### PR DESCRIPTION
**Pull request by xg-wang**
https://github.com/github/fetch/compare/141665b...master
2 changes worth notice due to fetch spec change:
https://github.com/github/fetch/pull/651 https://github.com/github/fetch/pull/640

### CODE REVIEW

File: pretender.js [hunk 1]
```diff
@@ -15,8 +15,7 @@ var FakeXMLHttpRequest = appearsBrowserified ? getModuleDefault(require('fake-xm
   self.FakeXMLHttpRequest;
 
 // fetch related ponyfills
-// TODO: use whatwg-fetch once new version release
-var FakeFetch = appearsBrowserified ? getModuleDefault(require('@xg-wang/whatwg-fetch')) : self.WHATWGFetch;
+var FakeFetch = appearsBrowserified ? getModuleDefault(require('whatwg-fetch')) : self.WHATWGFetch;
```
Comment (jherdman) [inline pretender.js]: I noted in #221 thinking that it's a documentation bug, but `self.WHATWGFetch` is undefined for me.

**Comment by xg-wang** [comment]
@rwjblue @stefanpenner need review

**Comment by xg-wang** [comment]
@stefanpenner Do you think we should check the import of `FakeFetch` to avoid the breaking upgrade like #221 ?

**Comment by xg-wang** [comment]
`yarn link "pretender"` on `ember-cli-mirage` and all tests pass.
So that we won't break pretender 1->2 upgrade. See https://github.com/samselikoff/ember-cli-mirage/pull/1389 for some usecase discussion.

**Comment by xg-wang** [comment]
Anything need to do to get this merged?

**Pull request merged**